release/pkg/assets/config/bundle_release.go

```diff
@@ -388,6 +388,15 @@ var bundleReleaseAssetsConfigMap = []assettypes.AssetConfig{
 			{
 				RepoName: "eks-anywhere-packages",
 			},
+			{
+				RepoName: "eks-anywhere-packages-bundles",
```

Comment:
Are package bundle k8s version-specific?

Reply:
Yes. Right now they are all the same, but we build on for each active version

Reply:
Okay, the last commit fixes the version specific problem